src/super_gradients/recipes/arch_params/unet_default_arch_params.yaml

```diff
@@ -0,0 +1,51 @@
+backbone_params:
+  in_channels: 3
+  strides_list: [2, 2, 2, 2, 2]   #  list of stride per stage.
+  width_list: [32, 64, 128, 256, 512]   # list of num channels per stage.
+  num_blocks_list: [2, 3, 3, 2, 2]    # list of num blocks per stage.
+  block_types_list: [REPVGG, REPVGG, REPVGG, REPVGG, REPVGG]   # list of block types per stage. See unet_encoder.DownBlockType for options.
+  is_out_feature_list: [ True, True, True, True, True ]   # list of flags whether stage features should be an output.
+  block_params:
+    anti_alias: True                # RepVGG stage param
+    droppath_prob: 0.               # XBlock stage param
+    bottleneck_ratio: 1.            # XBlock stage param
+    group_width: 16                 # XBlock stage param
+    se_ratio:                       # XBlock stage param
+    steps: 4                        # STDC stage params
+    stdc_downsample_mode: dw_conv   # STDC stage params
+
+
+context_module: aspp
```

Comment:
this should look like the following:
```
context_module: aspp
    dilation_list: [2, 4, 6]
    in_out_ratio: 1.
    use_bias: False
    in_channels:  512  # currently it is manually inserted as lats width of backbone 
```
you just need to use `resolve_paramter` (see my comment in unet.py)

Reply:
So you want in_channels to be manually hardcoded, and not dynamically adapted to the backbone?